lib/twitter_cldr/js/mustache/bundle.coffee

```diff
@@ -30,6 +30,12 @@ TwitterCldr.locale = "{{current_locale}}";
 {{> utilities}}
 {{{contents}}}
 
+TwitterCldr.set_data = (bundle) ->
+  for key, sub_bundle of bundle
+    for bundle_key, bundle_value of sub_bundle
+      @[key][bundle_key] = bundle_value
```

Comment:
With the way it's written now, to get things to work in the browser, you'd need to load the two files using script tags (in any order), and then doing

```javascript
TwitterCldr.setData(TwitterCldrDataBundle);
```

The data bundle gets loaded as `TwitterCldrDataBundle`. This it to make sure that there is no naming / variable conflict and you can load the two in any order and neither destructively overwrites the other.

---

Having said that, your suggestion does make the setup more simple and makes more sense overall. Even though it might add more overhead of variable name check at load time, the simplicity of setup probably is worth it. I'll give it a shot and push it in the next commit.